iceoryx_posh/test/moduletests/test_capro_message.cpp

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "iceoryx_posh/internal/capro/capro_message.hpp"
+#include "iceoryx_posh/internal/roudi/introspection/port_introspection.hpp"
+#include "test.hpp"
+
+using namespace ::testing;
+using namespace iox::capro;
+
+class CaproMessage_test : public Test
+{
+  public:
+    CaproMessage_test()
+    {
+    }
+
+    virtual ~CaproMessage_test()
+    {
+    }
+
+    virtual void SetUp(){};
+    virtual void TearDown(){};
+};
+
+
+TEST_F(CaproMessage_test, CTorSetsParametersCorrectly)
+{
+    uint16_t testServiceID = 1;
+    uint16_t testEventID = 2;
+    uint16_t testInstanceID = 3;
+    ServiceDescription sd = ServiceDescription(testServiceID, testEventID, testInstanceID);
```

Comment:
[Prefer direct Ctor call] Preferably use ```ServiceDescription sd(testServiceID, testEventID, testInstanceID);``` Another option is ```auto sd = ServiceDescription(testServiceID, testEventID, testInstanceID);```. All of these just call the Ctor, but the preferred option is the shortest.

Reply:
okay